lightning/src/ln/channel.rs

```diff
@@ -483,8 +483,8 @@ macro_rules! secp_check {
 
 impl<Signer: Sign> Channel<Signer> {
 	// Convert constants + channel value to limits:
-	fn get_holder_max_htlc_value_in_flight_msat(channel_value_satoshis: u64) -> u64 {
```

Comment:
Can you add a comment about this method "Returns a max in-flight inbound htlc value the remote can't overbid, required by us" ?